### PR TITLE
add firebase configuration

### PR DESCRIPTION
**Pull request by tjprange**
Database can send and recv data

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
-import React from 'react';
+import React, { useEffect } from 'react';
```
Comment (miladmikeal) [inline src/App.js:1]: Why is `useEffect` being imported here?

File: src/App.js [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
 import { registerRootComponent } from 'expo';
 import { NavigationContainer } from '@react-navigation/native';
-import TabNavigator from './navigation/TabNavigator';
-// eslint-disable-next-line import/no-unresolved
 import { TEST_VAR } from '@env';
```
Comment (miladmikeal) [inline src/App.js:4]: Also, you can delete this. I left it there so there was an example to go off.

File: config/firebase.config.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import * as firebase from 'firebase';
+import 'firebase/firestore';
+import {
+  TEST_VAR,
```
Comment (miladmikeal) [inline config/firebase.config.js:4]: Don't need to import TEST_VAR here.

File: src/App.js (hunk 1 above)
Comment (tjprange) [reply src/App.js:1]: Used it for getting data into the app. Will delete.

**Review by anderja8 — APPROVED** [review]
Looks good to me

**Pull request merged**